locust/main.py

```diff
@@ -210,7 +210,7 @@ def parse_options():
        action='store_true',
        dest='only_summary',
        default=False,
-       help='Only print the summary stats'
+       help="Only print the summary stats"
```

Comment:
super nitpicky but why this change?

Reply:
Looking back, I have no idea. I think it was to be consist with some code elsewhere.